src/containers/org/OrgDataSetContainer.js

```diff
@@ -93,6 +126,12 @@ const OrgDataSetContainer = ({
                 <Typography variant="h1">{title || name}</Typography>
                 <DataSetActionButton dataSetId={dataSetId} organizationId={organizationId} />
               </SpaceBetweenGrid>
+              <div>
+                <CountBadge count={dataSetColumns.size} />
+                <Label subtle>Data Fields</Label>
+                { dataSetSize && <CountBadge count={dataSetSize} max={500000} /> }
```

Comment:
When `dataSetSize` is 0:

<img width="266" alt="Screen Shot 2021-05-06 at 4 50 24 PM" src="https://user-images.githubusercontent.com/440299/117379889-5aaf6d00-ae8d-11eb-8536-bdcaa05ca4fb.png">